coderd/provisionerdserver/provisionerdserver.go

```diff
@@ -490,14 +490,14 @@ func (server *Server) UpdateJob(ctx context.Context, request *proto.UpdateJobReq
 				slog.F("stage", log.Stage),
 				slog.F("output", log.Output))
 		}
-		//nolint:gocritic // Provisionerd has specific authz rules.
-		logs, err := server.Database.InsertProvisionerJobLogs(dbauthz.AsProvisionerd(context.Background()), insertParams)
+
+		logs, err := server.Database.InsertProvisionerJobLogs(ctx, insertParams)
```

Comment:
Why did this get changed?

Reply:
I was looking through some other stuff at the same time. This is essentially a no-op so I just went ahead and removed it here.